src/navigation/pagination/text-pagination/PTextPagination.vue

```diff
@@ -47,6 +49,10 @@ export default {
             type: Boolean,
             default: true,
         },
+        isExistNextPage: {
+            type: Boolean,
+            default: true,
+        },
```

Comment:
Since this is UI related props, `disableNextPage` would be better.
```suggestion
        disableNextPage: {
            type: Boolean,
            default: false,
        },
```